src/catch2/internal/catch_main.cpp

```diff
@@ -33,7 +33,18 @@ int main (int argc, char * argv[]) {
     // and its constructor, as it (optionally) registers leak detector
     (void)&Catch::leakDetector;
 
-    return Catch::Session().run( argc, argv );
+    Catch::Session session;
+
+#if defined(CATCH_CONFIG_MAIN_BAZEL_JUNIT_XML_OUTPUT_FILE)
```

Comment:
That's a terrible define name. How about `CATCH_CONFIG_BAZEL_SUPPORT`? (Does Bazel have more of these customization env option?)

Also it needs to be documented in docs/configuration.md

Reply:
Agreed. I just sorta went for _something_. Happy to spin to `CATCH_CONFIG_BAZEL_SUPPORT` if we don't spin out another Bazel specific `main`.